modules/log/file.go

```diff
@@ -137,7 +137,7 @@ func (log *FileLogger) docheck(size int) {
 
 func (log *FileLogger) createLogFile() (*os.File, error) {
 	// Open the log file
-	return os.OpenFile(log.Filename, os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0o660)
+	return os.OpenFile(log.Filename, os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0o664)
```

Comment:
Some logs may contain sensitive information. 

If you need to make other process read Gitea's log, you can add that user to Gitea's user group.